src/components/ChannelHeader/ChannelHeader.tsx

```diff
@@ -15,6 +15,7 @@ import type {
   DefaultReactionType,
   DefaultUserType,
 } from '../../types/types';
+import { MenuIcon as DefaultMenuIcon } from './icons';
```

Comment:
This import should go above import { AvatarProps, Avatar as DefaultAvatar } from '../Avatar'; with a new line between it and surrounding imports. We group items by depth and abc.